src/sentry/models/user.py

```diff
@@ -392,6 +393,10 @@ def get_orgs(self):
 
         return Organization.objects.get_for_user_ids({self.id})
 
+    # A helper primarily here to make User/APIUser more compatible
+    def get_option(self, **kwargs):
```

Comment:
👍

Reply:
In some future, we can remove this once all code paths converge on `request.user` => `APIUser`